protocols/index/contracts/Index.sol

```diff
@@ -70,90 +71,90 @@ contract Index is Ownable {
+    * @param _data bytes32
     */
-  function setSignal(
+  function setLocator(
     address _user,
     uint256 _score,
-    bytes32 _locator
+    bytes32 _data
   ) external onlyOwner {
 
-    require(!hasSignal(_user), "USER_HAS_ENTRY");
+    require(!hasLocator(_user), "LOCATOR_ALREADY_SET");
 
-    Signal memory newSignal = Signal(_user, _score, _locator);
+    Locator memory newLocator = Locator(_user, _score, _data);
 
     // Insert after the next highest score on the linked list.
-    Signal memory nextSignal = findPosition(_score);
+    Locator memory nextLocator = findPosition(_score);
 
-    // Link the newSignal into place.
-    link(signalsLinkedList[nextSignal.user][PREV], newSignal);
-    link(newSignal, nextSignal);
+    // Link the newLocator into place.
+    link(locatorsLinkedList[nextLocator.user][PREV], newLocator);
+    link(newLocator, nextLocator);
 
     // Increment the length of the linked list if successful.
     length = length + 1;
 
-    emit SetSignal(_score, _user, _locator);
+    emit SetLocator(_score, _user, _data);
   }
 
   /**
-    * @notice Unset an Signal to Trade
+    * @notice Unset an Locator to Trade
     * @param _user address
     */
-  function unsetSignal(
+  function unsetLocator(
     address _user
   ) external onlyOwner returns (bool) {
 
     // Ensure the _user is in the linked list.
-    if (!hasSignal(_user)) {
+    if (!hasLocator(_user)) {
       return false;
     }
 
     // Link its neighbors together.
-    link(signalsLinkedList[_user][PREV], signalsLinkedList[_user][NEXT]);
+    link(locatorsLinkedList[_user][PREV], locatorsLinkedList[_user][NEXT]);
 
     // Delete user from the list.
-    delete signalsLinkedList[_user][PREV];
-    delete signalsLinkedList[_user][NEXT];
+    delete locatorsLinkedList[_user][PREV];
+    delete locatorsLinkedList[_user][NEXT];
 
     // Decrement the length of the linked list.
     length = length - 1;
 
-    emit UnsetSignal(_user);
+    emit UnsetLocator(_user);
     return true;
   }
 
   /**
-    * @notice Get the Signal for a user
+    * @notice Get the Locator for a user
     * @param _user address
     */
-  function getSignal(
+  function getLocator(
     address _user
-  ) external view returns (Signal memory) {
+  ) external view returns (Locator memory) {
 
     // Ensure the user has a neighbor in the linked list.
-    if (signalsLinkedList[_user][PREV].user != address(0)) {
+    if (locatorsLinkedList[_user][PREV].user != address(0)) {
 
-      // Return the next intent from the previous neighbor.
-      return signalsLinkedList[signalsLinkedList[_user][PREV].user][NEXT];
+      // Return the next Locator from the previous neighbor.
+      return locatorsLinkedList[locatorsLinkedList[_user][PREV].user][NEXT];
     }
-    return Signal(address(0), 0, bytes32(0));
+    return Locator(address(0), 0, bytes32(0));
   }
 
   /**
-    * @notice Get Valid Signals
+    * @notice Get Valid Locators
     * @param _count uint256
```

Comment:
may want an @return since return just the data component of the locator

Reply:
Makes sense.